notebooks/MAST/ultrafaint_dwarf_galaxy_search/ultrafaint_dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">from astroquery.mast import Mast
from astroquery.mast import Observations
from astroquery.mast import Catalogs
</pre><p>You can consolidate all three classes into one liner below.</p><p> </p><pre class="ql-syntax" spellcheck="false">from astroquery.mast import Mast, Observations, Catalogs
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>